src/lib/Canvas.ts

```diff
@@ -111,6 +120,7 @@ export class Canvas {
 	}
 
 	public set width(value: number) {
+		// @ts-expect-error: skia-canvas complains about 'width' being read-only.
```

Comment:
Report this to the team then

Reply:
It's [not read-only](https://github.com/samizdatco/skia-canvas/blob/6ea0f8bae2b8ea6f49308d4e7726278efa8b1f83/src/canvas.rs#L111-L133) in latest skia-canvas, however, types are provided by DefinitelyTyped, so you can guess the rest.

Reply:
This comment will get removed compile-time, so verify that this doesn't cause issues when not having `skipLibCheck` enabled by verifying the build output.

Reply:
It doesn't matter because code implementation isn't part of the typings.

What happens in the implementation, stays within the implementation.